src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIStructureWithWorkOrder.java

```diff
@@ -355,7 +356,7 @@ public void executeSpecificCompleteActions()
                     building.setBuildingLevel(woh.getUpgradeLevel());
                 }
             }
-            job.complete();
```

Comment:
Why the move of the job complete?

Reply:
![image](https://user-images.githubusercontent.com/37031914/59403740-c0098a80-8d58-11e9-906b-a638e3bd9b8d.png)

That needed to happen after job.complete(). You may ask Raycoms about it (he showed me it).

Reply:
Okey I am fine then